### PR TITLE
Prevent to open a second splitview from LuigiClient

### PR DESCRIPTION
**Pull request by JohannesDoberer**

### CODE REVIEW

File: core/src/App.html [hunk 1]
```diff
@@ -622,8 +622,10 @@
   //// SPLIT VIEW
   export const openSplitView = (nodepath, settings) => {
     if (mfSplitView.displayed) {
-      console.warn('Only one splitview can be oppened at a time');
+      console.warn('Only one splitview can be opened at a time');
       return;
+    } else {
+      mfSplitView = SplitViewSvc.getDefaultData().mfSplitView;
     }
     SplitViewSvc.open(getComponentWrapper(), nodepath, settings);
```
Comment (stanleychh) [inline core/src/App.html:630]:
```suggestion
    }
    mfSplitView = SplitViewSvc.getDefaultData().mfSplitView;
    SplitViewSvc.open(getComponentWrapper(), nodepath, settings);
```
Does this make sense and work properly?

Comment (JohannesDoberer) [reply core/src/App.html]: jap, good point :)

**Review by stanleychh — APPROVED** [review]
👍

**Review by ndricimrr — APPROVED** [review]
👍

**Pull request merged**